### PR TITLE
Add planck w/ git auto-update [author][new]

### PR DESCRIPTION
**Pull request by shakiba**
Pull request for issue: #
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/shakiba/planck.js
 * Official website (optional, not the repository): http://piqnt.com/planck.js/
 * NPM package url (optional): https://www.npmjs.com/package/planck-js
 * License and its reference: ZLIB
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 14
   - Count of stars: 400+
   - Count of forks: 12
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [ ] The parent of the commit(s) in the PR is not old than 3 days.
 * [ ] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/planck/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    "box2d",
+    "html5",
+    "javascript",
+    "game",
+    "physics",
+    "engine",
+    "2d",
+    "mobile"
+  ],
+  "license": "zlib",
```
Comment (extend1994) [inline ajax/libs/planck/package.json]: According to our [license list reference](https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json#L338), could you please modify this to "Zlib"? Thanks.

File: ajax/libs/planck/package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  },
+  "author": "Ali Shakiba",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/shakiba/planck.js.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "*"
```
Comment (extend1994) [inline ajax/libs/planck/package.json]: Let's use `**/*`.

**Comment by shakiba** [comment]
By the way, is `filename` required?

**Comment by shakiba** [comment]
@PeterDaveHello I find above communication offending, feel free to merge it when you are motivated enough to fix some typos or provide a reasonably easy to use system.

**Comment by PeterDaveHello** [comment]
@shakiba I'll send it for you.

**Comment by PeterDaveHello** [comment]
@shakiba should be good now, please feel free to help point problems like typos, we're happy to fix them.

**Pull request closed without merging**